added rpccorsdomain, logfile disabled for 1.4.0-unstable

euri10 opened the pull request:
sorry for the .gitignore in the same pull
https://github.com/pipermerriam/populus/issues/83

pipermerriam commented:
Can you add a new `click` command to the `chain_run` command something like the following so that it's configurable from the command line.

```python
@click.option(
     '--rpccorsdomain', default=None,
     help="""
     Determines the value that will be passed in as the `--rpcorsdomain` to the `geth` instance.
     """)
def chain_run(...):
    ...
```

pipermerriam commented on populus/cli/chain_cmd.py:
```diff
@@ -63,11 +63,12 @@ def chain_run(name, mine, verbosity, active):
-        "verbosity": "%d" % verbosity
+        # "logfile": logfile_path,
+        "verbosity": "%d" % verbosity,
+        "rpccorsdomain": 'http://localhost:5000'
```
Have you tested this?  I believe the `http://` should be left off but I can't recall if that is the case or not.

pipermerriam commented:
Also there are a few linting errors. https://travis-ci.org/pipermerriam/populus/jobs/124547131

pipermerriam commented:
Once tests are passing this looks great.

euri10 commented:
added the click, changed the tox, and yest the http:// works fine. 1st time I do this so please apologize if I'm messing things up, git is rather hard for newbs like me :)

euri10 commented:
I think this fails test because I commented # "logfile": logfile_path,

pipermerriam commented:
I appreciate the work you put into this.  Unfortunately, I don't believe that this pull request is viable given the significant changes that were just merged into master. 

If you would like to rework this to include the `--rpccorsdomain` changes to the `populus chain run` command I'd be happy to get that merged in.
